entity-service-impl/src/main/java/org/hypertrace/entity/data/service/EntityDataServiceImpl.java

```diff
@@ -108,6 +121,7 @@ public void upsert(Entity request, StreamObserver<Entity> responseObserver) {
           Entity.newBuilder(),
           entitiesCollection,
           responseObserver);
+      sendChangeNotification(tenantId, existingEntityCollection, List.of(normalizedEntity));
```

Comment:
should this move out of the try? we should never onError after we've already onComplete'd which I'm guessing is happening in upsertEntity. Same on all others. Even if we hadn't completed already, the rpc calls to ES should never fail due to the notifications of the changes. IMO, we should swallow and log all errors in the publisher and always call it as the last line of the success branch, after onComplete.

Reply:
As you have reviewed, this part of fixed.